modules/serverless-deprecated-removed-features.adoc

```diff
@@ -6,61 +6,44 @@
 [id="serverless-deprecated-removed-features_{context}"]
 = Deprecated and removed features
 
-Some features available in previous releases have been deprecated or removed.
-
-Deprecated functionality is still included in {ServerlessProductName} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed within {ServerlessProductName}, refer to the table below.
-
-In the table, features are marked with the following statuses:
-
-* *-*: _Not yet available_
-* *TP*: _Technology Preview_
-* *GA*: _General Availability_
-* *DEP*: _Deprecated_
-* *REM*: _Removed_
+Some features that were Generally Available (GA) or a Technology Preview (TP) in previous releases have been deprecated or removed. Deprecated functionality is still included in {ServerlessProductName} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed within {ServerlessProductName}, refer to the table below.
```

Comment:
s/the table below./the following table:/

Per IBM SG about indicating relative location in a document. Also colon instead of period.